Documentation/ExtensionArchitecture/Extbase/Reference/Validation/Validators/Index.rst

```diff
@@ -0,0 +1,624 @@
+:navigation-title: List of Validators
+
+..  include:: /Includes.rst.txt
+..  _extbase-validator:
+
+=======================================
+Built-in Validators provided by Extbase
+=======================================
+
+This document lists all built-in Extbase validators available in TYPO3 13,
+along with example usage for each using PHP attributes.
+
+..  seealso::
+    *   `Using validation for Extbase models and controllers <https://docs.typo3.org/permalink/t3coreapi:extbase-validation>`_
+    *   `Custom Extbase validator implementation <https://docs.typo3.org/permalink/t3coreapi:extbase-domain-validator>`_
+
+..  contents:: Validators in Extbase
+
+..  _extbase-validator-alpanumeric:
+
+AlphanumericValidator
+=====================
+
+Ensures a value contains only letters and numbers.
+
+..  code-block:: php
+
+    #[Validate(['validator' => 'Alphanumeric'])]
+    protected string $username;
+
+..  _extbase-validator-boolean:
+
+
+BooleanValidator
+================
+
+Checks if a value matches a specific boolean value (`true` or `false`).
+
+This validator is useful when you want to enforce that a property explicitly
+evaluates to either `true` or `false`, such as for checkboxes in forms.
+
+By default, it does not enforce any specific value unless the `is` option is
+set.
+
+**Options:**
+-   `is`: Accepts `'true'`, `'1'`, `'false'`, `'0'`, or `''`. Values are
+  converted internally to boolean `true` or `false`.
+
+Ensure that a value is a boolean (no strict check, default behavior):
+
+..  code-block:: php
+
+    #[Validate(['validator' => 'Boolean'])]
+    protected $isActive;
+
+Require that a value must be `true` (e.g. checkbox must be checked):
+
+..  code-block:: php
+
+    #[Validate(['validator' => 'Boolean', 'options' => ['is' => 'false']])]
```

Comment:
```suggestion
    #[Validate(['validator' => 'Boolean', 'options' => ['is' => true]])]
```

Wrong value, also I'd suggest to use real booleans by default here. Numbers/strings are only supported for other contexts.